gensim/test/test_ldamodel.py

```diff
@@ -293,19 +293,15 @@ def testTermTopics(self):
         numpy.random.seed(0)
         model = self.class_(self.corpus, id2word=dictionary, num_topics=2, passes=100)
 
+        # checks if topic '1' is in the result list
         # check with id 
         result = model.get_term_topics(2)
-        expected = [(1, 0.1066)]
-        # FIXME: fails on win and osx 
-        # self.assertEqual(result[0][0], expected[0][0])
-        # self.assertAlmostEqual(result[0][1], expected[0][1], places=2)
+        self.assertTrue(1 in result[0])
```

Comment:
Whether `1` (which is `topic_1`)  beats the probability cutoff and is returned along with a probability value.